extension/src/config/config.js

```diff
@@ -28,6 +48,14 @@ function getCtpConfig(ctpProjectKey) {
     authUrl:
       ctpConfig.authUrl || 'https://auth.europe-west1.gcp.commercetools.com',
   }
+  if (ctpConfig.authentication) {
+    result.authentication = {
+      scheme: ctpConfig.authentication?.scheme,
```

Comment:
You don't need optional checking in this case.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/2b89217e537adf47a97a5afe9cd604c28717a6be